styles/global.less

```diff
@@ -190,3 +190,8 @@
     font-weight: normal;
     font-style: oblique;
 }
+
+
+/* EB Garamond
+   https://fonts.google.com/specimen/EB+Garamond */
+@import url('https://fonts.googleapis.com/css2?family=EB+Garamond:wght@700&display=swap');
```

Comment:
I'm worried about external requests from css in case someone has their browser block 3rd party requests, especially from a google domain (to help prevent tracking). Does their license allow us to check it in like jquery?

Reply:
Even if it does allow it I don't want to check it in. If their browser blocks the load it will fall back to the browser default font. With jQuery, however, if their browser blocks it the site breaks, which is why I included it in the code.

In general I would prefer we embed less in the codebase, not more.

Reply:
That makes sense, just clarifying I suppose our philosophy here.

Reply:
Yup, totally fair question. 👍